packages/coding-standards/composer.json

```diff
@@ -12,9 +12,12 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-tokenizer": "*",
```

Comment:
newly required extensions should be described in documentation: `docs/introduction/required-php-configuration.md`